dns_stealth_tool.py

```
@@ -0,0 +1,715 @@
+        
+        col1, col2 = st.columns(2)
+        
+        with col1:
+            st.markdown("### 📝 Domain Information")
+            
+            domain_to_check = st.text_input("Domain to Check", "example.com")
+            
+            # خيارات الفحص
+            st.markdown("#### 🔧 Check Options")
+            
+            check_types = st.multiselect(
```

Ruff (0.12.2)

Local variable `check_types` is assigned to but never used

Remove assignment to unused variable `check_types`

(F841)